mlflow/store/dbmodels/db_types.py

```diff
@@ -0,0 +1,11 @@
+POSTGRES = 'postgresql'
+MYSQL = 'mysql'
+SQLITE = 'sqlite'
+MSSQL = 'mssql'
+
+DB_ENGINES = [
```

Comment:
Could you add a file-level comment indicating what this file contains (namely, the set of sqlalchemy database backend schemas that we support as tracking server backends).